river/cluster/clustream.py

```diff
@@ -45,6 +50,9 @@ class CluStream(base.Clusterer):
     time_window
         If the current time is `T` and the time window is `h`, we only consider
         the data that arrived within the period `(T-h,T)`.
+    time_gap
+        After each `time_gap`, the incremental K-Means will be applied on the current set of micro-clusters
```

Comment:
```suggestion
        An incremental k-means is applied on the current set of micro-clusters after each `time_gap`.
```

Reply:
We have decided to get rid of the use of `time_gap` from the discussion above. As such, this would be marked as resolved!